### PR TITLE
Add an example of using the Vault Open Service Broker

### PR DESCRIPTION
**Pull request by emaildanwilson**
This leverages the existing walkthrough configuration and shows how to setup and use the Vault Open Service Broker.

Overview
* Run Vault Dev on k8s
* Run the Vault OSB on k8s
* Create Broker for Vault OSB
* Use the Broker
* Clean Up

### CODE REVIEW

File: contrib/examples/vault/README.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
+# Service Catalog Vault Open Service Broker
+
+This document shows how to use the [Cloud Foundry Vault Service Broker](https://www.hashicorp.com/blog/cloud-foundry-vault-service-broker/) on Kubernetes by leveraging the Service Catalog. The Vault installation steps are not hardened for production use and are provided for ease of use only. See the [Vault docs](https://github.com/hashicorp/vault) for more information on how to properly configure a production grade service.
```
Comment (MHBauer) [inline contrib/examples/vault/README.md]: wrap long columns or one sentence per line please.

Comment (emaildanwilson) [inline contrib/examples/vault/README.md]: Thank you for the review! updated.

**Comment by duglin** [comment]
I think this might be more appropriate in this doc:  https://github.com/openservicebrokerapi/servicebroker/blob/master/gettingStarted.md   or at least a reference to it

**Comment by MHBauer** [comment]
I didn't even know that page existed. That's our "list of service brokers" that people should know about.

**Comment by emaildanwilson** [comment]
This seems to be more specific to kubernetes than what is described on that page. I'm ok with adding a link to it though.

If you'd rather not include specific examples of how to use the service catalog here then I can find somewhere else to post this. I'm just trying to help out.

**Comment by emaildanwilson** [comment]
Closing since the preferred method is to link to external examples.

**Pull request closed without merging**